set.mm

```diff
@@ -267117,18 +267113,17 @@ is based on direct sums of abelian groups ("We define on [the direct sum of
     $( Base set of the direct sum module using the ~ fndmin abbreviation.
        (Contributed by Stefan O'Rear, 1-Feb-2015.) $)
     dsmmbas2 $p |- ( ( R Fn I /\ I e. V ) -> B = ( Base ` ( S (+)m R ) ) ) $=
-      ( vx wfn wcel wa c0g cbs cfv crab wceq cvv c0 ccom cdif cdm cfn cdsmm wne
-      cv co cprds fveq2i rabeqi simpll fvco2 sylan neeq2d rabbidva eqid wn noel
-      reldmprds ovprc1 fveq2d base0 eleq2d mtbiri con4i adantl simplr prdsbasfn
-      syl6eqr simpr wf fn0g dffn2 mpbi biimpi fco sylancr ffnd ad2antrr fndmdif
-      syl2anc fndm rabeqdv 3eqtr4d eleq1d syl5eq fnex dsmmbase syl eqtrd ) CFKZ
-      FGLZMZAEUGZNCUAZUBUCZUDLZEBOPZQZDCUEUHOPZIWNWTJUGZWOPZXBCPNPZUFZJCUCZQZUD
-      LZEDCUIUHZOPZQZXAWNWTWREXJQXKWREWSXJBXIOHUJUKWNWRXHEXJWNWOXJLZMZWQXGUDXMX
-      CXBWPPZUFZJFQZXEJFQZWQXGXMXOXEJFXMXBFLZMXNXDXCXMWLXRXNXDRWLWMXLULZFNCXBUM
-      UNUOUPXMWOFKWPFKZWQXPRXMXJCDWOFSGXIXIUQXJUQXLDSLZWNYAXLYAURZXLWOTLWOUSYBX
-      JTWOYBXJTOPTYBXITODCUIUTVAVBVCVJVDVEVFVGWLWMXLVHXSWNXLVKVIWLXTWMXLWLFSWPW
-      LSSNVLZFSCVLZFSWPVLNSKYCVMSNVNVOWLYDFCVNVPFSSNCVQVRVSVTJFWOWPWAWBWLXGXQRW
-      MXLWLXEJXFFFCWCWDVTWEWFUPWGWNCSLXKXARFGCWHJXKCDESXKUQWIWJWKWG $.
+      ( vx wfn wcel wa c0g cfn cbs cfv crab wceq cvv cv ccom cdif cdm cdsmm wne
+      co cprds fveq2i rabeqi simpll fvco2 sylan neeq2d eqid reldmprds strov2rcl
+      rabbidva adantl simplr simpr prdsbasfn fn0g dffn2 mpbi biimpi fco sylancr
+      ffnd syl fndmdif syl2anc fndm rabeqdv 3eqtr4d eleq1d syl5eq fnex dsmmbase
+      wf eqtrd ) CFKZFGLZMZAEUAZNCUBZUCUDZOLZEBPQZRZDCUEUGPQZIWDWJJUAZWEQZWLCQN
+      QZUFZJCUDZRZOLZEDCUHUGZPQZRZWKWDWJWHEWTRXAWHEWIWTBWSPHUIUJWDWHWREWTWDWEWT
+      LZMZWGWQOXCWMWLWFQZUFZJFRZWOJFRZWGWQXCXEWOJFXCWLFLZMXDWNWMXCWBXHXDWNSWBWC
+      XBUKZFNCWLULUMUNURXCWEFKWFFKZWGXFSXCWTCDWEFTGWSWSUOZWTUOZXBDTLWDWTCWSUHDW
+      EXKXLUPUQUSWBWCXBUTXIWDXBVAVBXCWBXJXIWBFTWFWBTTNVTZFTCVTZFTWFVTNTKXMVCTNV
+      DVEWBXNFCVDVFFTTNCVGVHVIVJJFWEWFVKVLXCWBWQXGSXIWBWOJWPFFCVMVNVJVOVPURVQWD
+      CTLXAWKSFGCVRJXACDETXAUOVSVJWAVQ $.
   $}
```

Comment:
The proof size shrinks from 48 to 41 lines.  In the list of Syntax Definitions hints wn is not listed any longer.  This is explicable since tho operator NOT does not appear in the proof any more.  The list of Axioms and Definitions do not change.